src/packet/sym_encrypted_integrity_protected_data.js

```diff
@@ -54,28 +53,55 @@ class SymEncryptedIntegrityProtectedDataPacket {
   }
 
   constructor() {
-    this.version = VERSION;
+    this.version = 1;
```

Comment:
```suggestion
    this.version = null;
```
might have to be done in different PR